fixedwindow.go

```diff
@@ -126,3 +131,76 @@ func (f *FixedWindowRedis) Increment(ctx context.Context, window time.Time, ttl
 		return 0, ctx.Err()
 	}
 }
+
+// FixedWindowDynamoDB implements FixedWindow in DynamoDB.
+type FixedWindowDynamoDB struct {
+	client       *dynamodb.Client
+	partitionKey string
+	tableProps   DynamoDBTableProperties
+}
+
+// NewFixedWindowDynamoDB creates a new instance of FixedWindowDynamoDB.
+// PartitionKey is the key used to store all the this implementation in DynamoDB.
+//
+// TableProps describe the table that this backend should work with. This backend requires the following on the table:
+// * SortKey
+// * TTL
+func NewFixedWindowDynamoDB(client *dynamodb.Client, partitionKey string, props DynamoDBTableProperties) *FixedWindowDynamoDB {
+	return &FixedWindowDynamoDB{
+		client:       client,
+		partitionKey: partitionKey,
+		tableProps:   props,
+	}
+}
+
+const (
+	fixedWindowDynamoDBUpdateExpression = "SET #C = if_not_exists(#C, :def) + :inc, #TTL = :ttl"
+	dynamodbWindowCountKey              = "Count"
+)
+
+// Increment increments the window's counter in DynamoDB.
+func (f *FixedWindowDynamoDB) Increment(ctx context.Context, window time.Time, ttl time.Duration) (int64, error) {
+	var resp *dynamodb.UpdateItemOutput
+	var err error
+
+	done := make(chan struct{})
+	go func() {
+		defer close(done)
+		resp, err = f.client.UpdateItem(ctx, &dynamodb.UpdateItemInput{
+			Key: map[string]types.AttributeValue{
+				f.tableProps.PartitionKeyName: &types.AttributeValueMemberS{Value: f.partitionKey},
```

Comment:
Is it possible to use `strconv.FormatInt()` here and in other similar places throughout this PR?
If it is possible, please update the code accordingly

Reply:
done